webapp/src/containers/components/modals/deleteConfigModal.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useState} from 'react';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import './styles.css';
+
+interface Props {
+    visible: boolean;
+    setVisibility: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+function DeleteConfigModal(props: Props) {
```

Comment:
use `types` and arrow function everywhere

Reply:
i explored difference between them, any particular reason to use arrow function

Reply:
Arrow functions are preferred for their concise syntax, lexical `this` binding, and implicit return which are important.
You can explore in detail coding standards for ECMAScript.

Reply:
also `interface` is used where there is a need to extend or override them like in libraries but for custom components or components local to a project where props of one component are not going to be used another then it's advised to use `types`.
make these changes.